server/install.py

```diff
@@ -68,13 +65,14 @@ def create_worker_log_dir():
 
 
 def install_all_testers():
```

Comment:
Overall, I do not want you to remove the existing key stored in redis (`autotest:schema`). Indeed, a good demonstration that your changes are backwards compatible are to generate the exact same values in redis for this key as we previously had.